components/index/VolunteerHow.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <hr class="c-how__divider">
+    <div class="c-how">
+      <div class="c-how__section">
+        <img class="c-how__images" src="~/static/images/leds.svg" />
```

Comment:
It might be easier to do the correct alignment by using this as a background image of `c-how__individual` instead of an `img`. Makes it easier to position top left, and then all you have to do is add left padding to allow for it.